cmd/storage-rest-server.go

```diff
@@ -47,6 +48,23 @@ func (s *storageRESTServer) writeErrorResponse(w http.ResponseWriter, err error)
 	w.(http.Flusher).Flush()
 }
 
+type bulkErrorsResponse struct {
+	Errs []error `json:"errors"`
+}
+
+func (s *storageRESTServer) writeErrorsResponse(w http.ResponseWriter, errs []error) {
+	w.WriteHeader(http.StatusOK)
+	resp := bulkErrorsResponse{Errs: make([]error, len(errs))}
+	for idx, err := range errs {
+		if err == nil {
+			continue
+		}
+		resp.Errs[idx] = err
+	}
+	json.NewEncoder(w).Encode(resp) // TODO: check json encoding error
```

Comment:
We don't us json, we should use gob instead

Reply:
> We don't us json, we should use gob instead

Done